### ISSUE TITLE
Empty Folder omitted on load

### ISSUE THREAD

**Issue by SirSmurfalot** [MacTee/KSP-Mod-Admin-aOS#75]
I am using Ship Manifest and getting this error:

[Exception]: IsolatedStorageException: Could not find a part of the path "C:\Kerbal Space Program\GameData\ShipManifest\Plugins\PluginData\SMSettings.dat"

The zip for the module contains the folder "PluginData", but does not create the "SMSettings.dat" until your first launch (I am guessing)

It appears that MA is detecting "PluginData" as an empty folder and omitting it from the loadup, because the folder is not showing up in the MA mod tree or in the GameData for my KSP install.  As a result Ship Manifest cannot create the .dat file at that address and causes a bug.

**Comment by 4o66** [comment]
Can confirm, empty folders not processed.

**Comment by 4o66** [comment]
Pull request submitted to fix this issue.

**Comment by SirSmurfalot** [comment]
Good job on the speedy pull request. :+1: 

The ship manifest dev confirmed that file is not created until launch and I tested to confirm the missing folder was causing the file to not be created correctly.  He took the additional step of having his SM code autocheck for the missing folder and create it if necessary.  So it looks like this is going to be patched on both sides.  :)

I found at least one possibly two other mods throwing warnings because of something similar taking place with an empty folder, but that is not resulting in a game crash.
